pkg/migrations/op_change_type.go

```diff
@@ -138,5 +138,10 @@ func (o *OpChangeType) Validate(ctx context.Context, s *schema.Schema) error {
 	if o.Down == "" {
 		return FieldRequiredError{Name: "down"}
 	}
+
+	if err := validateName(o.Column); err != nil {
+		return err
+	}
+
```

Comment:
I don't think it makes sense to check the length of the column name on the operation here as it must already exist in the database. The check can be removed.